bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/TestUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.cerner.bunsen.avro;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+
+public class TestUtil {
+
+  /**
+   * This is used when we want to verify that no information is lost in the process of converting to
```

Comment:
This sounds fine as during the process we don't loose any information. May be we also file a separate issue to enhance the Avro record creation so that it contains the resourceType similar to the json representation, this way it is consistent across multiple formats.

Reply:
Agreed; I actually filed #1214 last week and added it to this comment now.